Celo-Identity.md

```diff
@@ -12,7 +12,7 @@ A barrier for the regular usage of a lot of blockchain systems in underserved ar
 
 Attestation means "evidence or proof of something". Celo has an Attestations Protocol which asks users to prove control over a phone number, which they can then link to their wallet address.
 
-Essentially what it is is an OTP-based service. You will receive a one-time passcode on your phone through SMS, and submit that OTP to the Attestation Service, through which you prove control over your phone number. Then, the service will link your phone number to your address.
+Essentially what it is an OTP-based service. You will receive a one-time passcode on your phone through SMS, and submit that OTP to the Attestation Service, through which you prove control over your phone number. Then, the service will link your phone number to your address.
```

Comment:
This shouldn't be removed. the double 'is' is meaningful here, not a typo

Reply:
Actually, I think it's better to say `Essentially, it is an OTP-based service` rather than `Essentially what it is is an OTP-based service`